content/docs/command-reference/data/status.md

```diff
@@ -43,32 +43,39 @@ DVC uncommitted changes:
 (there are other changes not tracked by dvc, use "git status" to see)
 ```
 
-As shown above, the `dvc data status` displays changes in multiple categories:
-
-- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
-  and `.dvc` files but the corresponding cache files are missing.
-- _DVC committed changes_ indicates that there are changes that are
-  `dvc-commit`-ed that differs with the last Git commit. There might be more
-  detailed state on how each of those files changed: _added_, _modified_,
-  _deleted_ and _unknown_.
-- _DVC uncommitted changes_ indicates that there are changes in the working
-  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
-  there might be more detailed state on how each of those files changed.
-- _Untracked files_ shows the files that are not being tracked by DVC and Git.
-  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
-  is specified.
-- _DVC unchanged files_ shows the files that are not changed. This is not shown
-  by default, unless [`--unchanged`](#--unchanged) is specified.
-
-By default, `dvc data status` does not show individual changes inside the
-tracked directories, which can be enabled with [`--granular`](#--granular)
-option.
+`dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that there are file hashes in `.dvc` or `dvc.lock`
+  files, but the corresponding <abbr>cache</abbr> files are missing. This may
+  happen after cloning a DVC repository but before using `dvc pull` (or
+  `dvc fetch`) to download the data; or after using `dvc gc`.
+
+- _DVC committed changes_ are new, modified, or deleted tracked files or
+  directories that have been [committed to DVC]. These may be ready for
+  committing to Git.
+
+- _DVC uncommitted changes_ are new, modified, or deleted tracked files or
+  directories that have not been [committed to DVC] yet. You can `dvc add` or
+  `dvc commit` these.
+
+- _Untracked files_ have not been added to DVC (nor Git). Only shown if the
+  `--untracked-files` flag is used.
+
+- _Unchanged files_ have no modifications. Only shown if the `--unchanged` flag
+  is used.
+
+Individual changes to files inside [tracked directories] are not shown by
+default but this can be enabled with the `--granular` flag.
+
+[committed to dvc]: /doc/command-reference/commit
+[tracked directories]: /doc/command-reference/add#adding-entire-directories
 
 ## Options
 
-- `--granular` - show granular, file-level information of the changes for
-  DVC-tracked directories. By default, `dvc data status` does not show
-  individual changes for files inside the tracked directories.
+- `--granular` - show granular file-level changes inside DVC-tracked
+  directories. Note that some granular changes may be reported as `unknown` as
+  DVC tracks
+  [directory-level hash values](doc/user-guide/project-structure/internal-files#directories).
```

Comment:
> I don't understand this comment tbh.
_From https://github.com/iterative/dvc.org/commit/aca9d8a1a170c6e818d93f26f545f93f866c14de#r83797370_

Do you mean you're not familiar with the underlying situation this refers to @shcheklein? Please see https://github.com/iterative/dvc.org/pull/3924#discussion_r966209543 if so. It's kind of confusing (maybe I didn't even get it right, waiting for another review from the core team).

Another alternative is to not even document this specifically, if it's evident what `unknown` means in the CLI output (self-documenting).

Reply:
Another alternative is to not even document this specifically, if it's evident what unknown means in the CLI output (self-documenting).

```suggestion
  directories.
```

Reply:
I might understand this if read the comment (i haven't yet), but I would expect to be able to understand it from the text, right?

do you mean that some `files (not granular changes) can be reported as "unknown" if (when? `as DVC tracks` is not informative)`?

Reply:
Yeah it's not super easy to explain briefly. How about this?
```suggestion
  directories. Note that granular changes may have `unknown` state if tracked
  [directory contents](doc/user-guide/project-structure/internal-files#directories)
  are not present in the local cache (see also `dvc fetch`).
```

Reply:
Status for files inside DVC-tracked directories may be reported as "unknown" if information about directories is missing from the cache yet (see ...)